src/ResourceManagement/Logic/Logic.Tests/InMemoryTests/IntegrationAccountSessionInMemoryTests.cs

```diff
@@ -0,0 +1,316 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+
+namespace Test.Azure.Management.Logic
+{
+    using System.IO;
+    using System.Linq;
+    using System.Net;
+    using System.Net.Http;
+    using Microsoft.Rest;
+    using Microsoft.Rest.Azure;
+    using Microsoft.Azure.Management.Logic.Models;
+    using Microsoft.Azure.Management.Logic;
+    using Xunit;
+
+    public class IntegrationAccountSessionInMemoryTests : InMemoryTestsBase
+    {
+        public IntegrationAccountSessionInMemoryTests()
+        {
+            var content = File.ReadAllText(path: @"TestData/IntegrationAccountSessionResponseContent.json");
```

Comment:
Because these are in-memory tests and not playback of scenario tests. This is the pattern used by all other APIs tests in this solution. I believe the intend of the original author was to be able to write a few tests without touching the service as basic validation before moving up to scenario tests.